rclcpp/test/test_function_traits.cpp

```diff
@@ -142,15 +143,15 @@ TEST(TestFunctionTraits, arity) {
       return 0;
     };
 
-  auto lambda_one_int = [](int) {
+  auto lambda_one_int = [](int /*one*/) {
```

Comment:
Is there a suggestion for how to address this differently and avoid the cpplint warnings?